src/utils/pythonHelper.mts

```diff
@@ -75,6 +79,17 @@ export default async function findPython(): Promise<string | undefined> {
             );
           }
         }
+        // if path does not exist, clear it
+        else {
+          Logger.warn(
+            LoggerSource.pythonHelper,
+            "Python path set in user settings does not exist:",
+            pythonPath
+          );
+          await Settings.getInstance()?.updateGlobal(
+            SettingsKey.python3Path,
+            undefined
+          );
```

Comment:
Could you fix the linting issue - I believe you're missing a closing brace here
```suggestion
          );
        }
```

Reply:
How did I miss that!